webapp.py

```diff
@@ -0,0 +1,92 @@
+from flask import Flask, render_template, request, redirect, url_for, make_response
+from main import RESULT_DIR, FOLDER_TYPES, TARGET_FOLDER
+import lib.arrange
+import lib.utils
+import webbrowser
+import time
+import os
+
+app = Flask(__name__)
+
+
+def gettheAddress():
+    """Read cache file and create target folder"""
+    temp = request.cookies.get('directory')
+    if temp is None:
+        address = TARGET_FOLDER
+    else:
+        address = temp
+    return address

```

Comment:
I don't understand this piece of code. Can you please explain this to me? I don't think setting it to TARGET_FOLDER will work? Right?

Reply:
This is checking the cookies, and if the user has not set any different address, the installation directory is used as the target address. I have also changed the variable name at function definition from TARGET_FOLDER to TARGET & TARGET_DIR, AS it raised some warnings.